### DISCUSSION TITLE
flinksql实现iotdbcdc的问题咨询

### DISCUSSION THREAD

**Discussion by fmiaojie** [apache/iotdb#13408] [Q&A] [answered]
怎么使用flinksql实现iotdb的cdc：
CREATE TABLE iotdb_table2 (
  Time_ BIGINT,
  `root.test.sn` INT,
  `root.test.T` DOUBLE,
  `root.test.x` DOUBLE,
  `root.test.y` DOUBLE
) WITH (
  'connector' = 'IoTDB',
  'nodeUrls' = '192.168.10.52:6667',
  'user' = 'root',
  'password' = 'root',
  'mode' = 'CDC',
  'cdc.task.name' = 'test',
  'sql' = 'SELECT Time_, sn, T, x, y FROM root.test',
  'cdc.pattern' = 'root.test'
);

CREATE TABLE JDBC_table4 (
  Time_ BIGINT,
  sn INT,
  temperature DOUBLE,
  x DOUBLE,
  y DOUBLE
) WITH (
  'connector' = 'jdbc',
  'url' = 'jdbc:mysql://192.168.61.128:3306/target?useSSL=false&serverTimezone=UTC&allowPublicKeyRetrieval=true',
  'table-name' = 'iotdb2',
  'username' = 'root',
  'password' = '@As200228'
);

INSERT INTO
   JDBC_table4
SELECT
  Time_,
  `root.test.sn` AS sn,
  `root.test.T` AS temperature,
  `root.test.x` AS x,
  `root.test.y` AS y
FROM
  iotdb_table2;

我使用以上语句，运行时没有报错，但是也没有进行数据传输

**Comment by CritasWang** [comment]
方便的话可以加入开源群，沟通会刚方便一点哈

![screenshot-20240911-165026](https://github.com/user-attachments/assets/f566a89d-8681-44ba-ac6b-3e0cf7edc811)

**Comment by CritasWang** [comment, marked as answer]
另外示例可以看 https://github.com/apache/iotdb-extras/tree/master/examples/flink-sql
